src/client/data-access/UlsLanguageTranslationRepository.ts

```diff
@@ -1,35 +1,18 @@
 import LanguageTranslationRepository from '@/common/data-access/LanguageTranslationRepository';
 import LanguageTranslations, { StringTMap } from '@/datamodel/LanguageTranslations';
-
-export type LanguageNameInUserLangCallback = ( languageCode: string ) => string;
+import MwWindow from '../MwWindow';
 
 export default class UlsLanguageTranslationRepository implements LanguageTranslationRepository {
 
-	private readonly languagesAtEntity: string[];
-	private readonly getLanguageNameInUserLang: LanguageNameInUserLangCallback;
-
-	public constructor(
-		getLanguageNameByCode: LanguageNameInUserLangCallback,
-		languagesAtEntity: string[],
-	) {
-		this.getLanguageNameInUserLang = getLanguageNameByCode;
-		this.languagesAtEntity = languagesAtEntity;
-	}
-
 	public getLanguagesInLanguage( inLanguage: string ): Promise<LanguageTranslations> {
 		return new Promise<LanguageTranslations>( ( resolve ) => {
 			resolve( {
-				[inLanguage ]: this.getLanguagesNamesInLanguage(),
+				[inLanguage]: this.getLanguagesNames(),
 			} as LanguageTranslations );
 		} );
 	}
 
-	private getLanguagesNamesInLanguage(): StringTMap<string> {
-		const languageNamesInUserLang = {} as StringTMap<string>;
-		this.languagesAtEntity.forEach( ( item: string ) => {
-			languageNamesInUserLang[ item ] = this.getLanguageNameInUserLang( item );
-		} );
-		return languageNamesInUserLang;
+	private getLanguagesNames(): StringTMap<string> {
+		return new ( ( window as MwWindow ).wb.WikibaseContentLanguages.prototype.constructor )().getAllPairs();
```

Comment:
The content languages are a service I see us reusing to implement the LANGUAGE_INIT action so I don't believe it would be premature to inject it.

Reply:
It is still missing in dev mode (`src/mockup-entry.ts`).

Reply:
done

Reply:
If we inject this in the entry point, we should do the same for https://github.com/wmde/wikibase-termbox/blob/master/src/client/data-access/EntityRepository.ts (maybe also https://github.com/wmde/wikibase-termbox/blob/master/src/client/init.ts),  because I like the idea that in the end is one clear way where MwWindow comes from, otherwise we spread this in n+1 ways.

Reply:
Done